### ISSUE TITLE
Latest release 3 days old?

### ISSUE THREAD

**Issue by P-T-I** [fkie-cad/nvd-json-data-feeds#6]
In the readme you state that a daily release is foreseen; the current release is 3 days old, where can I find the release from today?

**Comment by rhelmke** [comment]
Hello, good catch! One of our monitoring scripts failed.

I'm investigating and will fix ASAP.

**Comment by P-T-I** [comment]
Cool! Thanks!

**Comment by rhelmke** [comment]
Sooo, the VM that creates the nightly releases ran out of memory when creating the `cve-all` feed .. Seems that the initial `8GB` were a little bit optimistic for level 9 `xz` compression of the whole NVD.. and the raw dataset just grew over that memory limit during compression.

We allocated `64GB` to the worker and added some workflows in #7 to establish external monitoring that notifies us once there hasn't been a release or sync in the past 24hrs.

Finally, I manually issued a release run to verify. New packages should be available [now](https://github.com/fkie-cad/nvd-json-data-feeds/releases). The regular release timetable should be fixed now :-).

Cheers
